resources/definitions/GEEETECH_A10M.def.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version": 2,
+    "name": "GEEETECH A10M",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Amit L",
+        "manufacturer": "GEEETECH TECH.",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "GEEETECH_A10M_1",
+            "1": "GEEETECH_A10M_2"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "GEEETECH A10M" },
+        "machine_width": {
+            "default_value": 220
+        },
+        "machine_height": {
+            "default_value": 220
+        },
+        "machine_depth": {
+            "default_value": 220

```

Comment:
[This publication](http://www.geeetech.com/wiki/index.php/Geeetech_A10M_3D_printer) claims that the size is 220x220x260. Did you make it smaller intentionally or was that a mistake?